netbox/extras/dashboard/widgets.py

```diff
@@ -229,7 +229,13 @@ def render(self, request):
             htmx_url = reverse(viewname)
         except NoReverseMatch:
             htmx_url = None
-        if parameters := self.config.get('url_params'):
+        parameters = self.config.get('url_params')
+        if page_size := self.config.get('page_size'):
+            if not parameters:
+                parameters = {}
+            parameters['page_size'] = page_size
```

Comment:
We can clean this up a bit by passing `{}` as the default for null `url_params`.
```suggestion
        parameters = self.config.get('url_params', {})
        if page_size := self.config.get('page_size'):
          parameters['page_size'] = page_size
```